batch/batch/server/server.py

```diff
@@ -466,23 +467,23 @@ def __init__(self, batch_id, job_id, attributes, callback, userdata, user, alway
                 await self.set_state('Cancelled')
 
     async def cancel(self):
-        # Cancelled, Complete
+        # Cancelled, Error, Failed, Success
```

Comment:
I think the function is named well enough; the comment may not be necessary.

Reply:
removed